spec/lib/simp/cli/config/items/action/set_production_to_simp_action_spec.rb

```diff
@@ -0,0 +1,114 @@
+require 'simp/cli/config/items/action/set_production_to_simp_action'
+require_relative '../spec_helper'
+
+describe Simp::Cli::Config::Item::SetProductionToSimpAction do
+  before :each do
+    @ci            = Simp::Cli::Config::Item::SetProductionToSimpAction.new
+#    @ci.silent     = true # comment out this line if you want to see the messages generated
```

Comment:
I should have re-enabled all of those before checking in, although I do like the idea of an environment variable allowing us to selectively debug each class.  That would be a better option than constantly changing the code.